docs/source/basics.rst

```diff
@@ -140,8 +140,15 @@ the builtins contains a definition like this for ``chr``:
 
     def chr(code: int) -> str: ...
 
-In stubs we don't care about the function bodies, so we use an
-ellipsis instead. That ``...`` is three literal dots!
+The stub files, being for annotation-purposes only, don't include
+redundant information about a function. In the example above, the
+three dot literal (ie. ``...``) is used to make the stub files valid 
+python.  The pass statement can be used as an alternative to ``...``. 
```

Comment:
I'm not convinced the additional text here adds anything. It gives it a more formal style, but as Guido said, the PEP is the formal reference and these docs generally take an informal style.